.babelrc

```diff
@@ -1,19 +1,7 @@
 {
   "presets": [
-    "flow",
-    ["env", { "modules": false, "loose": true }]
+    "@babel/flow",
+    ["@babel/env", { "loose": true }]
   ],
-  "comments": false,
-  "env": {
-    "cjs": {
-      "plugins": [
-        "transform-es2015-modules-commonjs"
-      ]
-    },
-    "test": {
```

Comment:
is this no longer needed for jest?

Reply:
babel/env is used without `modules: false` which is internally applied by rollup-plugin-babel. So jest gets transpiled modules.

Reply:
https://github.com/rollup/rollup-plugin-babel#modules